pipelines.md

```diff
@@ -219,14 +219,16 @@ oc apply -f <application-stack-name>-pipeline-run.yaml
 
 <!--
 // =================================================================================================
-// Running pipelines from the command line for your custom built application stacks
+// Using stacks that are published to internal and private registries by pipelines
 // =================================================================================================
 -->
 
-## Running pipelines from the command line for your custom built application stacks
+## Using stacks that are published to internal and private registries by pipelines
 
-If you are publishing your application stack images to any other registry you can specify your custom registry when you initialize a stack by using the `--stack-registry` option
-in the `appsody init` command. Alternatively use the following process to configure the custom repository from which your pipelines pull the container images.
+If you are publishing your application stack images to any registry other than Docker hub you can specify your custom registry when you initialize a stack by using the `--stack-registry` option
+in the `appsody init` command.  This will update the stack name in the `.appsody-config.yaml` to include the registry information that will be consumed by the pipeline.  
```

Comment:
odd carriage return between "option" and "in"

Change
This will update
to
Specifying a custom registry updates

Change
that will be consumed
to
that is consumed